engine/graphics/src/wscript

```diff
@@ -48,10 +48,16 @@ def build(bld):
     obj.source = [x for x in obj.source if not x.name.endswith('graphics_glfw_wrappers.cpp')]
 
     if platform_supports_feature(bld.env.PLATFORM, 'opengl', {}):
+
+        opengl_defines = []
+        if platform_supports_feature(bld.env.PLATFORM, 'compute', {}):
+            opengl_defines = ['DM_HAVE_PLATFORM_COMPUTE_SUPPORT']
+
         obj = bld.stlib(features        = 'cxx',
-                        includes        = ['.', 'opengl'] + proto_cpp_includes,
+                        includes        = ['.', 'opengl', 'opengl/win32'] + proto_cpp_includes,
```

Comment:
Why should non-win32 platforms include a win32 folder?
Then it feels like something is a bit off?